src/main/java/uk/gov/mca/beacons/service/model/Beacon.java

```diff
@@ -0,0 +1,89 @@
+package uk.gov.mca.beacons.service.model;
+
+import java.time.LocalDate;
+import java.util.UUID;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "beacons")
+public class Beacon {
+
+  @Id
+  @GeneratedValue
+  private UUID id;
+
+  private String beaconType;
+  private String hexId;
+  private String manufacturer;
+  private String model;
+  private String serialNumber;
+  private LocalDate batteryExpiry;
```

Comment:
@Ninamma Question on these types: do you think we should be using `LocalDateTime` or ? See: https://docs.oracle.com/javase/8/docs/api/java/time/LocalDateTime.html which should store the time to nanosecond precision

See also: https://www.baeldung.com/jpa-java-time#after-java-8 guide on persisting using JPA, should probably specify the column type as well which is `timestamp`.

See the GDS guide on using timestamps in services: https://www.gov.uk/government/publications/open-standards-for-government/date-times-and-time-stamps-standard

Reply:
@Ninamma As agreed, for now `LocalDate` should give us the precision we need.  I will capture a ticket on the GitHub Issues to monitor this going forwards.  We're storing them as `timestamp` in the DB so updating to use `LocalDateTime` should just be a case of updating the Java object and not the database column as it already storing them in `LocalDateTime` format.